frontend/frontend.sh

```diff
@@ -4,12 +4,13 @@ COMPOSE_FILE="docker-compose.yaml"
 
 build_frontend() {
     echo "Building frontend Docker service..."
-    docker-compose -f $COMPOSE_FILE build frontend
+    docker-compose  -f $COMPOSE_FILE build frontend --no-cache
```

Comment:
--no-cache has to be before after build and before frontend to work
docker-compose  -f $COMPOSE_FILE build --no-cache frontend